docs/source/tutorial.rst

```diff
@@ -7,47 +7,240 @@ Tutorial
+           audio_path_stereo="user/mir_datasets/orchset/audio/stereo/Beethoven-S3-I-ex1.wav",
+           composer="Beethoven",
+           contains_brass=False,
+           contains_strings=True,
+           contains_winds=True,
+           excerpt="1",
+           melody_path="user/mir_datasets/orchset/GT/Beethoven-S3-I-ex1.mel",
+           only_brass=False,
+           only_strings=False,
+           only_winds=False,
+           predominant_melodic_instruments=['strings', 'winds'],
+           track_id="Beethoven-S3-I-ex1",
+           work="S3-I",
+           audio_mono: (np.ndarray, float),
+           audio_stereo: (np.ndarray, float),
+           melody: F0Data,
+        )
+
+
+
+We can access to specific tracks by id. The ids are specified in the dataset index.
+In the next example we take the first track of the index, and then we retrieve the melody
+annotation.
+
+.. code-block:: python
+
+    orchset_ids = orchset.track_ids  # Load list of track ids of the dataset
+    orchset_data = orchset.load_tracks()  # Load dataset tracks
+    example_track = orchset_data[orchset_ids[0]]  # Get first track of the index
+
+    # Accessing to track melody annotation
+    example_melody = example_track.melody
+
+
+Alternatively, we don't need to load the whole dataset to get a single track.
+
+.. code-block:: python
+
+    orchset_ids = orchset.track_ids  # Load list of track ids of the dataset
+    example_melody = orchset.track(orchset_ids[0]).melody  # Get melody from first track in the index
+
+
+Annotations can also be accessed through ``load_someAnnotation()`` methods.
+
+.. code-block:: python
+
+    orchset_ids = orchset.track_ids  # Load list of track ids of the dataset
+    example_melody_path = orchset.track(orchset_ids[0]).melody_path  # Parsing melody annotation path
+
+    # Accessing to track melody annotation
+    example_melody = orchset.load_melody(example_melody_path)
+    print(example_melody.frequencies)
+    >>> array([  0.   ,   0.   ,   0.   , ..., 391.995, 391.995, 391.995])
+    print(example_melody.times)
+    >>> array([0.000e+00, 1.000e-02, 2.000e-02, ..., 1.244e+01, 1.245e+01, 1.246e+01])
+
+
+Annotation classes
+^^^^^^^^^^^^^^^^^^
+
+``mirdata`` uses several diffent data classes to store annotations. These data classes are meant to standarize the organization for
+all the loaders, and keep compatibility with `JAMS <https://jams.readthedocs.io/en/stable/>`_ and `mir_eval <https://craffel.github.io/mir_eval/>`_.
+
+The list of available annotation classes are:
+
+- BeatData(times, positions)
+- SectionData(intervals, labels)
+- NoteData(intervals, notes, confidence)
+- ChordData(intervals, labels, confidence)
+- F0Data(times, frequencies, confidence)
+- MultiF0Data(times, frequency_list, confidence_list)
+- KeyData(intervals, keys)
+- LyricData(intervals, lyrics, pronunciations)
+- TempoData(intervals, value, confidence)
+- EventData(intervals, events)
```

Comment:
For easier maintenance of the docs I'd simply include a link to the `annotations` sections in the docs, so in the future when we expand annotations we don't need to edit this one